packages/editor/src/components/post-title/index.js

```diff
@@ -63,6 +69,14 @@ export default function PostTitle() {
 		};
 	}, [] );
 
+	useImperativeHandle( forwardedRef, () => ( {
+		focusTitle: () => {
```

Comment:
Should this be just "focus"? it's already the "PostTitle" component so maybe focusTitle is a bit redundant

Reply:
Changed.